profile/src/main/java/org/openedx/profile/presentation/manage_account/compose/ManageAccountView.kt

```diff
@@ -0,0 +1,267 @@
+                                            .fillMaxWidth(),
+                                        text = stringResource(id = org.openedx.profile.R.string.profile_edit_profile),
+                                        onClick = {
+                                            onAction(ManageAccountViewAction.EditAccountClick)
+                                        },
+                                        borderColor = MaterialTheme.appColors.buttonBackground,
+                                        textColor = MaterialTheme.appColors.textAccent
+                                    )
+                                    Spacer(modifier = Modifier.height(12.dp))
+                                    IconText(
+                                        text = stringResource(id = org.openedx.profile.R.string.profile_delete_profile),
+                                        painter = painterResource(id = org.openedx.profile.R.drawable.profile_ic_trash),
+                                        textStyle = MaterialTheme.appTypography.labelLarge,
+                                        color = MaterialTheme.appColors.error,
+                                        onClick = {
+                                            onAction(ManageAccountViewAction.DeleteAccount)
+                                        })
+                                    Spacer(modifier = Modifier.height(12.dp))
+                                }
+                            }
+                        }
+                    }
+                    PullRefreshIndicator(
+                        refreshing,
+                        pullRefreshState,
+                        Modifier.align(Alignment.TopCenter)
+                    )
+                }
+            }
+        }
+    }
+}
+
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_NO)
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Preview(name = "NEXUS_5_Light", device = Devices.NEXUS_5, uiMode = Configuration.UI_MODE_NIGHT_NO)
+@Preview(name = "NEXUS_5_Dark", device = Devices.NEXUS_5, uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Composable
+private fun ManageAccountViewPreview() {
+    OpenEdXTheme {
+        ManageAccountView(
+            windowSize = WindowSize(WindowType.Compact, WindowType.Compact),
+            uiState = mockUiState,
+            uiMessage = null,
+            refreshing = false,
+            onAction = {}
+        )
+    }
+}
+
+
+@Preview(name = "NEXUS_9_Light", device = Devices.NEXUS_9, uiMode = Configuration.UI_MODE_NIGHT_NO)
+@Preview(name = "NEXUS_9_Dark", device = Devices.NEXUS_9, uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Composable
+private fun ManageAccountViewTabletPreview() {
+    OpenEdXTheme {
+        ManageAccountView(
+            windowSize = WindowSize(WindowType.Medium, WindowType.Medium),
+            uiState = mockUiState,
+            uiMessage = null,
+            refreshing = false,
+            onAction = {}
+        )
+    }
+}
+
+val mockAccount = Account(
```

Comment:
Can we make it private?

Reply:
We use this code in multiple files, so I relocated it to the ProfileUi